build/configs/rtl8721csm/security_hal_test/defconfig

```diff
@@ -130,7 +130,7 @@ CONFIG_ARCH_HAVE_RAMVECTORS=y
 #
 # Board Settings
 #
-CONFIG_BOARD_LOOPSPERMSEC=16717
+CONFIG_BOARD_LOOPSPERMSEC=22222
```

Comment:
This folder was renamed to `security_hal_test_tz` by PR #4385 . Could you rebase this to resolve conflict?

Reply:
Is there any more comment on the PR? Shall I do the rebase (it will take sometime) or just modify this particular conflict and force push on my last commit?

Reply:
Nomore comment. You can resolve it as you want

Reply:
I resolved the conflict with [2a7fbc4]